src/Uno.UI.RemoteControl.VS/EntryPoint.cs

```diff
@@ -419,6 +422,68 @@ async Task Restart()
 		}
 	}
 
+	private async Task OnInfoBarNotificationRequestedAsync(object? sender, NotificationRequestIdeMessage message)
+	{
+		try
+		{
+			await _asyncPackage.JoinableTaskFactory.SwitchToMainThreadAsync();
+
+			if (await _asyncPackage.GetServiceAsync(typeof(SVsShell)) is IVsShell shell &&
+				await _asyncPackage.GetServiceAsync(typeof(SVsInfoBarUIFactory)) is IVsInfoBarUIFactory infoBarFactory)
+			{
+				await CreateInfoBarFactoryAsync(message, shell, infoBarFactory);
+			}
+		}
+		catch (Exception e) when (_ideChannelClient is not null)
+		{
+			_debugAction?.Invoke($"Failed to handle InfoBar Notification Requested with message {e.Message}");
+			throw;
+		}
+	}
+
+	private async Task CreateInfoBarFactoryAsync(NotificationRequestIdeMessage e, IVsShell shell, IVsInfoBarUIFactory infoBarFactory)
+	{
+		if (_ideChannelClient is null)
+		{
+			return;
+		}
+		var factory = new InfoBarFactory(infoBarFactory, shell);
+
+		var infoBar = await factory.CreateAsync(
+			new InfoBarModel(
+				e.Message,
+				e.Commands.Select(Commands => new ActionBarItem
+				{
+					Text = Commands.Text,
+					Name = Commands.Name,
+					ActionContext = Commands.Parameter,
+					IsButton = true,
+				}).ToArray(),
+				e.Kind == NotificationKind.Information ? KnownMonikers.StatusInformation : KnownMonikers.StatusError,
+				isCloseButtonVisible: true));
+
+		if (infoBar is not null)
+		{
+			infoBar.ActionItemClicked += (s, e) =>
+			{
+				_asyncPackage.JoinableTaskFactory.Run(async () =>
+				{
+					if (e.ActionItem is ActionBarItem action &&
+						action.Name is { } command)
+					{
+						var cmd =
+							new CommandRequestIdeMessage(
+								System.Diagnostics.Process.GetCurrentProcess().Id,
+								command,
+								action.ActionContext?.ToString());
+						await _ideChannelClient.SendToDevServerAsync(cmd, _ct.Token);
+					}
+				});
```

Comment:
Using Process ID

Reply:
That's fine for now, please rename the property accordingly then (cc. @jeromelaban )